torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -1133,8 +1133,38 @@ def generate_c_shim_extern_kernel_call(self, kernel, args):
         # ArrayRefTensor and at::Tensor is still fragile.
         self.allow_stack_allocation = False
 
+        # HACK: val_to_arg_str jams multiple arguments together using a comma. If that
+        # ever breaks, it needs to be reworked to be able to return multiple arguments,
+        # and the split-on-comma code here needs to be removed.
+        def is_number(s: str):
+            try:
+                int(s)
+                return True
+            except ValueError:
+                pass
+
+            try:
+                float(s)
+                return True
+            except ValueError:
+                return False
+
+        wrapped_args = []
+        for x in args:
+            pieces = x.split(", ")
+            for piece in pieces:
+                # We only really *need* convert_arrayref_tensor_to_tensor for
+                # ArrayRefTensors. The code flowing into here uses `0` for nullptr,
+                # which convert_arrayref_tensor_to_tensor would blindly coerce to int,
+                # so just avoid wrapping integers.
+                if not is_number(piece):
```

Comment:
If ```0``` is the only case that would cause the issue, could we just skip such a case by simply ```if pieces != '0'```?

Reply:
No, it's not just 0, see the error msg in https://github.com/pytorch/pytorch/issues/123174

Reply:
So, is it safe to say that any non-number argument will be a tensor?